bigquery/query.go

```diff
@@ -149,6 +149,9 @@ type QueryConfig struct {
 	// Experimental: this option is experimental and may be modified or removed in future versions,
 	// regardless of any other documented package stability guarantees.
 	JobTimeout time.Duration
+
+	// Force usage of Storage API if client is available.
+	ForceStorageAPI bool
```

Comment:
Should this be in QueryConfig, or should it be on Client?  Alternatively, should we add this as a variadic option on our Read() func in tables/jobs/queries?